packages/ember-data/lib/system/relationships/belongs-to.js

```diff
@@ -89,19 +89,21 @@ function belongsTo(modelName, options) {
 
   return computedPolyfill({
     get: function(key) {
-      return this._relationships[key].getRecord();
+      return this._internalModel._relationships[key].getRecord();
```

Comment:
this pattern is also used in the tests, It seems pretty brittle and looks like it would anger the law of demeter